components/ws-manager/pkg/manager/testdata/cdwp_class.golden

```diff
@@ -8,7 +8,7 @@
                 "app": "gitpod",
                 "component": "workspace",
                 "gitpod.io/networkpolicy": "default",
-                "gitpod.io/workspaceClass": "foobar",
+                "gitpod.io/workspaceClass": "gitpod.io/internal/foobar",
```

Comment:
this is not a valid label value

Reply:
now I get it - sorry for that. I was under the impression that label values merely had a size limit, but was unaware of the character limitation 🤦

Reply:
I think we should use https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L167 to ensure we are setting a valid value

Reply:
💯 will add